pytest.ini

```diff
@@ -4,3 +4,256 @@ python_files=test_*.py __init__.py
+    scrapy/pipelines/media.py E125 E501 E266
+    # scrapy/selector
+    scrapy/selector/__init__.py F403 F401
+    scrapy/selector/unified.py F401 E501 E111
+    # scrapy/settings
+    scrapy/settings/__init__.py E501
+    scrapy/settings/default_settings.py E501 E261 E114 E116 E226
+    scrapy/settings/deprecated.py E501
+    # scrapy/spidermiddlewares
+    scrapy/spidermiddlewares/httperror.py E501
+    scrapy/spidermiddlewares/offsite.py E501
+    scrapy/spidermiddlewares/referer.py F401 E501 E129 W503 W504
+    scrapy/spidermiddlewares/urllength.py E501
+    # scrapy/spiders
+    scrapy/spiders/__init__.py F401 E501 E402
+    scrapy/spiders/crawl.py E501
+    scrapy/spiders/feed.py E501 E261 W391
+    scrapy/spiders/init.py W391
+    scrapy/spiders/sitemap.py E501
+    # scrapy/utils
+    scrapy/utils/benchserver.py E501
+    scrapy/utils/boto.py F401
+    scrapy/utils/conf.py E402 E502 E501
+    scrapy/utils/console.py E302 E261 F401 E306 E305
+    scrapy/utils/curl.py F401
+    scrapy/utils/datatypes.py E501 E226
+    scrapy/utils/decorators.py E501 E302
+    scrapy/utils/defer.py E501 E302 E128
+    scrapy/utils/deprecate.py E128 E501 E127 E502
+    scrapy/utils/display.py E302
+    scrapy/utils/engine.py F401 E261 E302
+    scrapy/utils/ftp.py E302
+    scrapy/utils/gz.py E305 E501 E302 W504
+    scrapy/utils/http.py F403 F401 W391 E226
+    scrapy/utils/httpobj.py E302 E501
+    scrapy/utils/iterators.py E501 E701
+    scrapy/utils/job.py E302
+    scrapy/utils/log.py E128 W503
+    scrapy/utils/markup.py F403 F401 W292
+    scrapy/utils/misc.py E501 E226
+    scrapy/utils/multipart.py F403 F401 W292
+    scrapy/utils/project.py E501
+    scrapy/utils/python.py E501 E302
+    scrapy/utils/reactor.py E302 E226
+    scrapy/utils/reqser.py E501
+    scrapy/utils/request.py E302 E127 E501
+    scrapy/utils/response.py E501 E302 E128
+    scrapy/utils/signal.py E501 E128
+    scrapy/utils/sitemap.py E501
+    scrapy/utils/spider.py E271 E302 E501
+    scrapy/utils/ssl.py E501
+    scrapy/utils/template.py E302
+    scrapy/utils/test.py E302 E501
+    scrapy/utils/url.py E501 F403 F401 E128 F405
+    # scrapy
+    scrapy/__init__.py E402 E501
+    scrapy/_monkeypatches.py W293
+    scrapy/cmdline.py E502 E501
+    scrapy/crawler.py E501
+    scrapy/dupefilters.py E302 E501 E202
+    scrapy/exceptions.py E302 E501
+    scrapy/exporters.py E501 E261 E226
+    scrapy/extension.py E302
+    scrapy/interfaces.py E302 E501 W391
+    scrapy/item.py E501 E128
+    scrapy/link.py E501 W391
+    scrapy/logformatter.py E501 W293
+    scrapy/mail.py E402 E128 E501 E502
+    scrapy/middleware.py E502 E128 E501
+    scrapy/pqueues.py E501
+    scrapy/resolver.py E302
+    scrapy/responsetypes.py E128 E501 E305
+    scrapy/robotstxt.py E302 E501
+    scrapy/shell.py E501
+    scrapy/signalmanager.py E501
+    scrapy/spiderloader.py E225 F841 E501 E126
+    scrapy/squeues.py E128
+    scrapy/statscollectors.py E501 W391
+    # tests
```

Comment:
Shouldn’t tests be excluded from these checks?

Reply:
hmmm I'm don't see any reason to do that, could you explain me why? :thinking: 

From my point of view, fixing the PEP8/style in the tests will contribute to their maintainability.